app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,27 @@
 class Api::BaseController < ApplicationController
   layout nil
-end
+
+  # POST /api/validate_email
+  def validate_email
+    email = params[:email]
+    if email_valid?(email)
```

Comment:
The `email_valid?` method uses a simple regex for email validation. Consider using a more comprehensive email validation library to handle edge cases and ensure better accuracy.